### DISCUSSION TITLE
Evaluate counter display ?

### DISCUSSION THREAD

**Discussion by mymeyer** [jomjol/AI-on-the-edge-device#1200] [Q&A]
Hello,
how can I evaluate such an water meter. With the unit 10 liters there are no numbers.
![Zählerbild](https://user-images.githubusercontent.com/44463914/197205259-f544845e-3d83-465f-88c8-6755c9f6fa7e.png)

Do you have an idea ?

MyMeyer

**Comment by caco3** [comment]
I don't think its an electricity mete, unless you pay per m3 for your current :smile: 

Did you try the detection of the 10l pointer? As far as I know, it doesn't depend on the numbers but on the pointer position, thus should work without any issue!

**Reply by mymeyer** [reply]
Oh yes right of course "water meter" :-) . 

I will test it. Thanks for your tip.

**Comment by ghost** [comment]
Not all analog pointers have numbers on it. The model recognize it in my opinion not on the numbers. The shadow could be a problem, but I would try it. If it not recognize it with good accuracy you could use https://github.com/haverland/collectmeteranalog the collect a dataset for training and sent it to us. (or put it here in).

Alternativly you could set extendedResolution=true.
